AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Organizations/OrganizationIdQuery.cs

```diff
@@ -1,10 +0,0 @@
-﻿using AllReady.Models;
-using MediatR;
-
-namespace AllReady.Areas.Admin.Features.Organizations
-{
-    public class OrganizationIdQuery : IAsyncRequest<int>
-    {
-        public int? ItineraryId { get; set; }
```

Comment:
@mgmccarthy I should probably explain what my intentions where for this query and handler as it looks like you've adjusted/renamed? I was thinking we could extend the query message to have nullable ints for all main lookups (itineraryid, eventid etc) and then a single handler which queries based on the id with a value. Thinking it through, separate specific queries and handlers might be slightly simplier and limited to single responsibility, although we could end up with many very repetitive handler, which if a domain model change occurs; all would need updating.

My intension with the handler was to get to a single initial query and just applying the appropriate where clause in the handler as required. I didn't get that far with it to date.